src/main/resources/normalization/marc4ld_subfield_rules.yml

```diff
@@ -0,0 +1,266 @@
+'subfieldRules':
+  '020$c':
+    - " :"
+    - ":"
+  '100$0':
+    - "."
+  '100$2':
+    - ","
+    - "."
+  '100$3':
+    - ","
+    - "."
+  '100$4':
+    - ","
+    - "."
+  '100$5':
+    - ","
+    - "."
+  '100$c':
+    - ","
+  '100$d':
+    - ","
+  '110$2':
+    - ","
+    - "."
+  '110$3':
+    - ","
+    - "."
+  '110$4':
+    - ","
+    - "."
+  '110$5':
+    - ","
+    - "."
+  '110$b':
+    - "."
+  '111$2':
+    - ","
+    - "."
+  '111$3':
+    - ","
+    - "."
+  '111$4':
+    - ","
+    - "."
+  '111$5':
+    - ","
+    - "."
+  '111$c':
+    - " :"
+  '111$d':
+    - ","
+  '111$q':
+    - "."
+  '130$0':
+    - ","
+    - "."
+  '130$2':
+    - ","
+    - "."
+  '130$3':
+    - ","
+    - "."
+  '130$5':
+    - ","
+    - "."
+  '130$c':
+    - ","
+  '130$f':
+    - "."
+  '130$g':
+    - "."
+  '130$l':
+    - "."
+  '130$n':
+    - ","
+    - "."
+  '130$p':
+    - ","
+    - "."
+  '130$s':
+    - "."
+  '240$f':
+    - "."
+  '240$g':
+    - "."
+  '240$l':
+    - "."
+  '240$n':
+    - ","
+    - "."
+  '240$p':
+    - ","
+    - "."
+  '240$s':
+    - "."
+  '242$y':
+    - "."
+  '245$b':
+    - " :"
+    - " ="
+  '245$c':
+    - " /"
+  '245$f':
+    - ","
+  '245$h':
+    - " /"
+  '245$n':
+    - "."
+    - ","
+  '245$p':
+    - "."
+    - ","
+  '246$a':
+    - ":"
+  '246$b':
+    - " :"
+    - " ="
+  '246$n':
+    - "."
+  '246$p':
+    - "."
+    - ","
+  '246$y':
+    - "."
+  '260$a':
+    - " ;"
+    - ","
+    - " :"
+  '260$b':
+    - " ;"
+    - " :"
+    - ","
+    - ";"
+    - " :"
+    - " "
+  '260$c':
+    - ","
+    - " :"
+    - ";"
+  '260$e':
+    - " :"
+  '260$f':
+    - " :"
+  '260$g':
+    - " :"
+  '264$3':
+    - " :"
+  '264$a':
+    - " ;"
+    - ","
+    - " :"
+  '264$b':
+    - " ;"
+    - " :"
+    - ","
+    - ";"
+    - " :"
+    - " "
+  '264$c':
+    - ","
+    - " :"
+    - ";"
+  '300$b':
+    - " :"
+  '300$c':
+    - " ;"
+  '310$b':
+    - ","
+  '505$g':
+    - " --"
+  '505$r':
+    - " /"
+  '505$t':
+    - " -"
+    - " --"
+    - " -- "
+    - " ;"
+  '510$c':
+    - ","
+  '600$0':
+    - "."
+  '600$c':
+    - ","
+  '600$d':
+    - ","
+  '600$l':
+    - "."
+  '600$t':
+    - "."
+  '610$0':
+    - "."
+  '610$b':
+    - "."
+  '611$0':
+    - "."
+  '611$c':
+    - " :"
+  '611$d':
+    - "."
+    - " :"
+  '6XX$2':
+    - "."
+    - ","
+  '6XX$c':
```

Comment:
`6XX$c` was not there in the original list given in ticket. right?

Reply:
Right PK, it wasn't there, removed it by the last commit. Double-checked the mappings, look correct now.